decoders/connector/lansitec/macro-bluetooth-gateway/v1.0.0/payload.js

```diff
@@ -0,0 +1,315 @@
+// BluetoothGateway
```

Comment:
New decoders that are added must be made in TypeScript and must have a unit test.